Neos.Neos/Classes/Command/UserCommandController.php

```diff
@@ -19,6 +19,7 @@
 use Neos\Flow\Security\Account;
 use Neos\Flow\Security\Exception\NoSuchRoleException;
 use Neos\Flow\Security\Policy\Role;
+use Neos\Party\Domain\Model\ElectronicAddress;
```

Comment:
Neos.Neos does _not_ have a dependency on `neos/party` its just installed because of the base dist.
Thats why this is kinda illegal :P Maybe a command controller inside neos/party?